src/dymaptic.GeoBlazor.Core/Components/Widgets/HomeWidget.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using System.Text.Json.Serialization;
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets;
+
+public class HomeWidget : Widget
+{
+    public override string WidgetType => "home";
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+
+    public string? IconClass { get; set; }
+    [Parameter]
+    public string? Label { get; set; } = "Home";
```

Comment:
I changed my mind about setting a default here, since the default for the widget should really be "no label" as in the JS samples.

Reply:
corrected and noted